project_restorer/src/main/scala/MessageSchema.scala

```diff
@@ -0,0 +1,3 @@
+case class MessageSchema(projectId: Int, numberOfAssociatedFiles: Int) {
```

Comment:
Braces are not needed here

Reply:
Would also be nicer to name it something more descriptive! - something like `RestorerSummaryMessage` ?